.circleci/config.yml

```diff
@@ -52,16 +52,23 @@ commands:
 jobs:
 ######################## PR OR MERGE TO STAGING ########################
   build_image:
-    <<: *default_machine_config
+    machine: # Use an image that has at least Docker 18.09 installed
+      image: ubuntu-1604:201903-01
     steps:
       - checkout
       - run:
           name: Build image
           command: |
-            docker login --username ${DOCKERHUB_USER} --password ${DOCKERHUB_PASSWORD} &&
-            export IMAGE_TAG=$([[ "$CIRCLE_BRANCH" == "staging" ]] && echo "staging-candidate" || echo "discardable") &&
-            IMAGE_NAME_CANDIDATE=snyk/kubernetes-monitor:${IMAGE_TAG}-${CIRCLE_SHA1} &&
-            ./scripts/docker/build-image.sh ${IMAGE_NAME_CANDIDATE} &&
+            docker login --username "${CONTAINER_REGISTRY_USERNAME}" --password "${CONTAINER_REGISTRY_PASSWORD}" registry.redhat.io
+            IMAGE_TAG=$([[ "$CIRCLE_BRANCH" == "staging" ]] && echo "staging-candidate" || echo "discardable")
+            IMAGE_NAME_CANDIDATE=snyk/kubernetes-monitor:${IMAGE_TAG}-${CIRCLE_SHA1}
+            RED_HAT_AUTH_CREDENTIALS_FILE=credentials
```

Comment:
can we push this image to a snyk org and use it from there? so we do this manually one time, and the local build and CI/CD stays basically the same

Reply:
I think it's not relevant any more